_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -0,0 +1,319 @@
+  where each participant has enough balance or capacity in their channel
+  to support the payment.  But if we use this mechanism when Alice has
+  10 open channels each containing a maximum of 10% of her total hot
+  wallet balance, Alice can only spend at most 10% of her funds at a
+  time.
+
+    Multipath payments provide a solution to this problem: if Alice
+    wants to send 15% of her funds, she can send 7.5% to Charlie through
+    her channel with Bob and 7.5% through her channel with Dan (who also
+    has a channel with Charlie).  Although this increases the complexity
+    of implementations somewhat due to having to deal with some partial
+    payments failing some of the time, it doesn't change the security as
+    all payments can use the normal LN security mechanism of releasing a
+    pre-image in order to claim payments (in this case, claiming
+    multiple payments with the same preimage rather than just one).
+
+    For more information, see the following Lightning-Dev threads
+    which often call this feature Atomic
+    Multi-path Payments (AMP): [1][roasbeef amp], [2][zmn base amp],
+    [3][pickhardt local amp].
+
+- **Dual-funded channels:** a nice feature of the current implementation
+  is that only one party to the channel needs to initially commit any
+  funds to it.  For example, Alice opens a channel to Bob with 0.1 BTC
+  of her money and none of Bob's money.  This makes it very easy for
+  users to accept new incoming channels as they don't cost the receiving
+  user anything---not even any opportunity costs (except for operating
+  the LN software).  However, this also means that channels can only be
+  used in one direction initially---Alice can pay Bob or route payments
+  through Bob, but Alice can't receive payments from Bob or from any
+  routing path including Bob until Alice has sent Bob some money.  This
+  creates a bootstrapping problem: if Alice wants to receive payments
+  via LN, she has to get people to open new channels to her node---which
+  requires paying possibly-high onchain fees and waiting for onchain
+  confirmation that can take hours.
+
+    A proposed solution to this problem is to allow dual-funded
+    channels.  Alice agrees to put 0.1 BTC into a channel with Bob if
+    Bob agrees to open the channel with 0.1 BTC of his own funds.  This
+    can cost Bob money---namely onchain transaction fees and lost
+    opportunity cost from having his funds committed for some time---but
+    Bob also receives the opportunity to earn LN routing fees for any
+    payments sent to Alice.
+
+    The basic implementation of dual-funding is probably simple (LN
+    nodes already handle bidirectional payments) but creating an
+    incentive mechanism that can reward capital providers like Bob is
+    still being discussed.  For more information, see the following
+    threads: [1][neigut liquidity], [2][zmn liquidity], [3][zmn dual
+    rbf].  See also [Newsletter #21][].
```

Comment:
I'm adding a note for "the initiator of the channel currently also pays the fees for closing the channel" as you suggest, but I don't see any other easy changes---it seems to me, as is said in the post, that finding the best incentive mechanism for dual-funded is still being discussed.  (One of the nice things about having a weekly newsletter, is that we often get the opportunity to write follow-up stories---so there's usually no need to go too deep on things that are still being worked on.  For example, this particular week's newsletter adds detail to items mentioned in three previous weeks.)

Is the symmetric CSV comment directly regarding dual-funded or is it a suggestion for a separate item?  I see https://github.com/lightningnetwork/lightning-rfc/wiki/Brainstorming#symmetric-csv-delay which does sound interesting and I see that it's one of the accepted proposals in the 1.1 spec draft.  I didn't mention CSV here since it wasn't in Rusty's highlights I used as a filter, but I'm happy to write it up if you think it's noteworthy (I just need to know whether it should go with dual-funded or be a separate item).